function.php

```diff
@@ -0,0 +1,37 @@
+<?php
+function getDBConnection()
+{
+    $db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+$db = getDBConnection();
```

Comment:
Don't call this in here, this file should only contain function definitions, not any code that actually runs. This line should be in your index.php instead